### ISSUE TITLE
Support nginx

### ISSUE THREAD

**Issue by neidiom** [systemli/ansible-role-roundcube#32]
I am getting this error

```

RUNNING HANDLER [systemli.roundcube : Restart apache] *************************************************************************************************************************************************************
fatal: [xxxx]: FAILED! => {"changed": false, "msg": "Could not find the requested service apache2: host"}
```

**Comment by neidiom** [comment]
https://github.com/systemli/ansible-role-roundcube/pull/33

**Comment by doobry-systemli** [comment]
Fixed by #33
